adoc/extensions/sycl_khr_graph.adoc

```diff
@@ -1,14 +1,29 @@
-= sycl_khr_command_graph
+= sycl_khr_graph
+
+== Overview
+
+The graph abstraction defines an explicitly lazy execution model by decoupling
+command creation from submission.
+These concepts are potentially tied together in SYCL implementations that
+eagerly submit work to a device when a command-group is submitted to a queue.
+By providing a prior construction stage before a workload of commands begins
+execution on the device, submission of multiple commands can happen in a single
+submission call instead of many, reducing the overhead of multiple independent
+command submissions.
+Deferral of command submission can also allow for the SYCL implementation to
+optimize dependencies within the user defined graph, which may improve
+concurrency and other performance metrics.
 
 == Dependencies
 
 This extension has no dependencies on other extensions.
 
-== Feature test macro
+== Contributors
+
+== Feature Test Macro
```

Comment:
I think we discussed this offline but didn't actually comment here, the SYCL spec seems to only put a capital on the first word of each section title, so I think actually `Feature test macro` was correct and should be standardized across all the titles in the spec.